src/main/resources/main.css

```diff
@@ -458,28 +486,6 @@ VBox.spaced, HBox.spaced {
     -fx-border-width: 0 0 1px 0;
 }
 
-.hyperlink {
```

Comment:
where has this been used before? seems like there are links in the gui now that are no more visible as links...